.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -0,0 +1,69 @@
+name: ecosystem-ci trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  trigger-all:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && startsWith(github.event.comment.body, '/ecosystem-ci run')
+    steps:
+      - uses: actions/github-script@v6
+        id: trigger
+        with:
+          # this token needs "repo" scope
+          github-token: ${{ secrets.ECOSYSTEM_CI_TRIGGER_GITHUB_PAT }}
+          result-encoding: string
+          script: |
+            console.log(`Get PR info: ${context.repo.owner}/${context.repo.repo}#${context.issue.number}`)
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            })
+
+            const startDate = new Date()
+
+            console.log(`Trigger ${context.repo.owner}/vite-ecosystem-ci ecosystem-ci.yml`)
+            await github.rest.actions.createWorkflowDispatch({
+              owner: context.repo.owner,
+              repo: 'vite-ecosystem-ci',
+              workflow_id: 'ecosystem-ci.yml',
+              ref: 'main',
+              inputs: {
+                refType: 'branch',
+                ref: pr.head.ref,
+                repo: pr.head.repo.full_name
+              }
+            })
+
+            await new Promise(resolve => setTimeout(resolve, 10 * 1000))
+
+            console.log(`Get lastest workflow run`)
+            // assumes the first one is the one triggered
+            const { data: runs } = await github.rest.actions.listWorkflowRuns({
+              owner: context.repo.owner,
+              repo: 'vite-ecosystem-ci',
+              workflow_id: 'ecosystem-ci.yml',
+              event: 'workflow_dispatch',
+              created: `>${startDate.toISOString()}`,
+              per_page: 1
+            })
```

Comment:
There isn't a way to get which workflow run is the triggered one. The URL might be wrong when many workflow runs were triggered at the same time.

Reply:
Possible workaround: https://stackoverflow.com/a/69500478